releasenotes-builder/src/main/java/com/github/checkstyle/Main.java

```diff
@@ -53,36 +54,37 @@ private Main() {
      * @noinspectionreason CallToSystemExit - main method must exit with code
      */
     public static void main(String... args) {
-        int errorCounter;
         List<String> publicationErrors = null;
+        final List<String> errors = new ArrayList<>();
+        boolean hasThrownException = false;
         try {
             final CliProcessor cliProcessor = new CliProcessor(args);
             cliProcessor.process();
             if (cliProcessor.hasErrors()) {
-                printListOf(cliProcessor.getErrorMessages());
-                errorCounter = cliProcessor.getErrorMessages().size();
+                errors.addAll(cliProcessor.getErrorMessages());
             }
             else {
                 final CliOptions cliOptions = cliProcessor.getCliOptions();
                 final Result notesBuilderResult = runGithubNotesBuilder(cliOptions);
+                errors.addAll(notesBuilderResult.getWarningMessages());
```

Comment:
Warnings shouldn't be errors, otherwise, the name should change.

Reply:
Would renaming to `errorsAndWarnings` be better in your opinion?

Reply:
It depends on how you plan to use it. If your using it to count errors, then no, as warnings as not errors. We shouldn't fail the utility when a warning is printed.

Why can't we keep them separate like how it was done before?

Reply:
You have a point, warnings have to be printed as before since we do not fail if there are such.

Reply:
Done. Reverted back the warnings.